deploy/rbac.yaml

```diff
@@ -18,9 +18,15 @@ metadata:
   labels:
     name: litmus
 rules:
-- apiGroups: ["","apps","batch","litmuschaos.io","apps.openshift.io"]
-  resources: ["pods","jobs","deployments","replicationcontrollers","replicasets","daemonsets","statefulsets","deploymentconfigs","events","configmaps","services","secrets","chaosengines","chaosexperiments","chaosresults"]
-  verbs: ["get","create","update","patch","delete","list","watch","deletecollection"]
+- apiGroups: ["","apps","batch","apps.openshift.io"]
+  resources: ["jobs","deployments","replicationcontrollers","daemonsets","replicasets","statefulsets","deploymentconfigs","events","services","secrets"]
+  verbs: ["get","list","watch"]
+- apiGroups: ["","litmuschaos.io"]
+  resources: ["pods","configmaps","chaosengines","chaosexperiments","chaosresults"]
+  verbs: ["get","create","update","delete","list","watch","deletecollection"]  
+- apiGroups: ["admissionregistration.k8s.io"]
+  resources: ["validatingwebhookconfigurations"]
```

Comment:
lets figure out what we really need here. Operator itself doesn't use any validation webhooks. There is a separate deploy for this controller. We are adding some permissions here for that controller to work using the same svcAcc.